### DISCUSSION TITLE
Hey, I was looking to enhance the docs a bit.

### DISCUSSION THREAD

**Discussion by bhfish2020** [backstagephp/filament-2fa#14] [Q&A]
The "Then add the plugin to your PanelProvider:" is a little vague, the `PanelProvider` is in the filament src but the published panel you can easily edit is in the `app/Providers/Filament/AdminPanelProvider`. Also on vs code it will not pick up the namespace of the class. Maybe adding the name space on the top of some of these would be helpful, Id be happy to fork and give it a shot if you're cool with it.

Also it seemed to require me to add the 'vendor/vormkracht10/filament-2fa/resources/dist/filament-two-factor-auth.css' to my vite.config.js file and run `npm run build` after

**Comment by bhfish2020** [comment]
It was a very fluid experience though! I can tell y'all worked really hard on it. This package deserves more love

**Comment by Baspa** [comment]
Hey @bhfish2020 feel free to submit a PR! We welcome any improvements to the package. If you have any suggestions to improve anything, like the documentation, go ahead 😄
